wiki/Help_centre/Account_restrictions/en.md

```diff
@@ -109,7 +117,9 @@ Please understand that sending an email to this address only guarantees a second
 | Multi-account | Permanent (multis) | Main account will be left unrestricted, possibly silenced. |
 | Excessive multi-accounting | 3 months | Appeal terms apply, 1 year tournament ban |
 | Account sharing | 3 months | Appeal terms apply, 1 year tournament ban, and partial rollback |
-| Cheating | 3 months | Appeal terms apply, 1 year tournament ban, and full rollback |
+| Subsequent offences (multi-accounting, account sharing, community conduct) | 6 months | Appeal terms apply, rollback (where appropriate), tournament ban 1 year per offence |
+| Cheating | 6 months | Appeal terms apply, 1 year tournament ban, and full rollback |
 | Excessive misconduct | Variable | Appeal terms apply. Cooldown will depend on history. |
-| Cheating/sharing for a tournament | 6 months | Appeal terms apply, appropriate rollback, permanent tournament ban |
-| Subsequent offences | 6 months | Appeal terms apply, appropriate rollback, tournament ban 1 year per offence |
+| Cheating/sharing for a tournament | 12 months | Appeal terms apply, appropriate rollback, permanent tournament ban |
+| Subsequent offences (cheating) | 12 months | Appeal terms apply, rollback, tournament ban 1 year per offence |
```

Comment:
if it can go to 24, it should be mentioned

```suggestion
| Subsequent offences (cheating) | 12 to 24 months | Appeal terms apply, rollback, tournament ban 1 year per offence |
```